code/socialDistribution/views.py

```diff
@@ -841,7 +842,18 @@ def post_comment(request, author_id, post_id):
                     "object": post.public_id
                 }
 
-                request_url = f'{post.author.strip("/")}/inbox/'
+                post_host = url_parser.get_host(post.public_id)
+                if post_host != HOST:
+                    request_url = post.public_id.strip("/") + '/comments/'
+                    
+                    if post_host == REMOTE_NODES["t16"]:
+                        data["id"] = post.public_id
+
+                    elif post_host == REMOTE_NODES["t11"]:
```

Comment:
can you pls also add "t20" as they also expect it at //service/author/{author_id}/posts/{post_id}/comments

Reply:
Sounds good
Will add them
Thanks! 🙏

Reply:
Actually, the code was sending it to the comments route for every team. One issue that I found though was that we need put the post's public_id in an id field of our payload for every team except team 11, which expects it inside an api_url field for some reason.